.github/ISSUE_TEMPLATE/Bug_Currency_Dashboard.yml

```diff
@@ -0,0 +1,39 @@
+name: Bug for Currency Dashboard
+title: "[BUG]: "
+description: "Bug reporting for currency dashboard"
+labels: "dashboard bug"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug or feedback reports for Currency Dashboard. Also to request currency for any package or report currency issues for any package.
+        Please fill in the following details to provide bug details for Currency Dashboard.
```

Comment:
"Please fill in the following details to help us reproduce the bug" sounds better?

Reply:
Again, I like Priya's version - "For issues with the Currency Dashboard, please give us as much information, including screen shots when possible to help us reproduce your issue or better understand the problem.  However, remember that any submitted information is public so do NOT include secrets, passwords, keys, PI, SPI, etc."

Reply:
Keeping combination of both suggested version
![image](https://github.com/ppc64le/build-scripts/assets/82815888/d0a0dc3b-7615-4a45-8789-c2ec062f24b5)